backend/controllers/emailDomainValidation.ts

```diff
@@ -1,5 +1,5 @@
 const isValidDomainNameChars = (domainName: string) => domainName.match(/^[0-9A-Za-z-,.()'&]*$/)
-const isValidDomainDescriptionChars = (domainDesc: string) => domainDesc.match(/^[0-9A-Z_]*$/)
+const isValidDomainDescriptionChars = (domainDesc: string) => domainDesc.match(/^[0-9A-Z_ ]*$/)
```

Comment:
Is additional space required ? [0-9A-Z_ ]

Reply:
Yes, I have updated the regex to /^[0-9A-Z_\s+]*$/